temporian/core/operators/arithmetic/__init__.py

```diff
@@ -0,0 +1,18 @@
+# pylint: disable=unused-import
+
+from temporian.core.operators.arithmetic.base import Resolution
+from temporian.core.operators.arithmetic.addition import sum, AdditionOperator
+from temporian.core.operators.arithmetic.division import (
+    divide,
+    DivisionOperator,
+)
+from temporian.core.operators.arithmetic.subtraction import (
+    subtract,
+    SubtractionOperator,
+)
+from temporian.core.operators.arithmetic.multiplication import (
+    multiply,
+    MultiplicationOperator,
+)
+
+# pylint: enable=unused-import
```

Comment:
I thought that all `__init__.py` files should be empty? cc: @achoum

Reply:
It's related with what I commented in the PR description. Maybe it's something to discuss.